web/src/components/base/CustomIconButton.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled'
+import { IconButton, IconButtonProps, SvgIconProps } from '@mui/material'
+import React, { ComponentType, ReactElement } from 'react'
+
+type CustomIconButtonProps = IconButtonProps &
+  React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+    icon: string | ComponentType<SvgIconProps>
+    ariaLabel: string
+  }
+
+const StyledIconButton = styled(IconButton)(() => ({
+  '& svg': {
+    width: 24,
+    height: 24,
+  },
+}))
```

Comment:
🔧 We should not manually adjust the sizes like this, instead, please pass the `size` prop as specified in the [docs](https://mui.com/material-ui/react-button/#sizes-2).